RELEASING.md

```diff
@@ -0,0 +1,138 @@
+<!-- markdownlint-configure-file { "MD013": { "line_length": 120 } } -->
+# Releasing
+
+## Overview
+
+This repo contains three types of release artifacts:
+
+* **Go Binaries** for Terraform Engine, Policy Generator, and Terraform Importer.
+* **Templates** for the Terraform Engine ("templates" for short).
+* **Policy templates** for the Policy Generator ("policies" for short).
+
+These three groups of artifacts are not coupled with each other, so they are **released separately**.
+
+### Versioning
+
+Releases are tagged with [Git Annotated Tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) following [semantic
+versioning](https://semver.org/) (semver):
+
+* `v1.x.x` - MAJOR version, increments on breaking changes.
```

Comment:
Can we potentially list some examples of what we consider breaking changes?
i.e. is pinning to a new version of Terraform considered a breaking change, or are we only considering breaking changes to the resources/functionality entirely contained within our package?

Reply:
I added some examples. Let me know if they're clear, or if I should add more.

Reply:
LGTM